tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -197,6 +197,12 @@ def test_brand_string(test_microvm_with_api):
         assert False
 
 
+# The X2APIC MSRs cannot to be loaded and stored on VMX transitions. A VMX transition fails
+# if the VMM has specified that the transition should access any MSRs in the address range
+# from 0000_0800H to 0000_08FFH
+X2APIC_MSRS = [hex(i) for i in range(0x0000_0800, 0x0000_08FF)]
```

Comment:
is this correct to exclude `0x0000_08FF` from the range?

Reply:
No it was wrong, fixed to include `0x0000_08FF`.